contracts/truefi2/TrueAssuranceFund.sol

```diff
@@ -5,14 +5,23 @@ import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
 import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
 import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
+import {Initializable} from "../common/Initializable.sol";
 import {ILoanToken2} from "./interface/ILoanToken2.sol";
 import {ITrueFiPool2} from "./interface/ITrueFiPool2.sol";
+import {ILoanFactory2} from "./interface/ILoanFactory2.sol";
 
-contract TrueAssuranceFund {
+contract TrueAssuranceFund is Initializable {
```

Comment:
Make UpgradeableOwnable

Reply:
To clarify, do you mean UpgradeableClaimable?

Reply:
Would be nice to have a consensus on that 😁
@hal909